python_modules/dagit/dagit/starlette.py

```diff
@@ -134,6 +134,7 @@ def _static_file(file_path):
             return Route(
                 file_path,
                 lambda _: FileResponse(path=self.relative_path(f"webapp/build{file_path}")),
+                name="root_static",
```

Comment:
Feels weird for multiple routes to have the same name - but I guess we're not using `url_for` for this, so it's fine as an identifier for now